mlir/lib/Dialect/SparseTensor/Transforms/SparseBufferRewriting.cpp

```diff
@@ -605,22 +608,22 @@ static void createPartitionFunc(OpBuilder &builder, ModuleOp module,
   Value i = lo;
   Value j = builder.create<arith::SubIOp>(loc, hi, c1);
   createChoosePivot(builder, module, func, nx, ny, isCoo, i, j, p, args);
-  SmallVector<Value, 3> operands{i, j, p}; // Exactly three values.
-  SmallVector<Type, 3> types{i.getType(), j.getType(), p.getType()};
+  Value cont = constantI1(builder, loc, true);   // The value for while (true)
```

Comment:
slightly better name than cont? Maybe just "t" for true, or trueVal if you want longer names